app/src/main-process/menu/build-default-menu.ts

```diff
@@ -546,7 +547,25 @@ export function buildDefaultMenu({
     showLogsItem,
   ]
 
-  if (__DEV__) {
+  if (__RELEASE_CHANNEL__ === 'development' || __RELEASE_CHANNEL__ === 'test') {
```

Comment:
Could we use `enableTestMenuItems` here for more clarity?
```suggestion
  if (enableTestMenuItems()) {
```